x-pack/plugins/monitoring/public/directives/elasticsearch/ml_job_listing/index.js

```diff
@@ -83,13 +117,24 @@ uiModule.directive('monitoringMlListing', kbnUrl => {
       const getNoDataMessage = filterText => {
         if (filterText) {
           return (
-            `There are no Machine Learning Jobs that match the filter [${filterText.trim()}] or the time range.
-Try changing the filter or time range.`
+            i18n('xpack.monitoring.elasticsearch.mlJobListing.noFilteredJobsDescription', {
+              // eslint-disable-next-line max-len
+              defaultMessage: 'There are no Machine Learning Jobs that match the filter [{filterText}] or the time range. Try changing the filter or time range.',
```

Comment:
Please try to use backticks to make the string multiline.

Reply:
We can't use backticks to make the string multiline, because the symbol of new line '/n' is inserted on UI and in our i18n JSON file.